packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/buttonLabel.st

```diff
@@ -0,0 +1,6 @@
+user interface
+buttonLabel
+	"Returns a matching label for the start/stop button depending if profiler is currently profilings or not."
```

Comment:
```suggestion
```

I’d suggest removing this comment as it explains what’s literally written in code in the next three lines.